packages/dts-generator/lib/phases/json-to-ast.js

```diff
@@ -11,51 +11,75 @@ function jsonToAst(jsonObj) {
 /**
  * @param symbols
  * @param {string} nsFQN
+ * @param {boolean} [isClass=false]
  * @returns {Namespace}
  */
-function buildNamespace(symbols, nsFQN) {
+function buildNamespace(symbols, nsFQN, isClass) {
```

Comment:
Should we have two functions instead of a boolean flag
with multiple conditionals?

Reply:
Done